source/components/molecules/InfoModal/InfoModal.stories.js

```diff
@@ -0,0 +1,92 @@
+
+ 
+
+När du loggar in i appen första gången hämtar vi personlig information om dig, så som namn, personnummer och adress från Skatteverket och ber dig att bekräfta att denna information är korrekt. Om uppgifterna inte stämmer måste du ta kontakt med Skatteverket. Vill du komplettera med ytterligare information, så som e-postadress eller telefonnummer, gör du det genom att fylla i din personliga profil i appen. Den information som vi hämtar från Skatteverket tillsammans med de uppgifter du själv lämnar i appen kommer att föreslås i de e-tjänster som du använder i appen, för att underlätta din användning av våra tjänster. 
+
+ 
+
+Utöver den personliga information som anges ovan kan vi komma att automatiskt samla in och behandla följande information när du använder appen:
+  + Teknisk information inklusive ID för mobil, enhetstyp och valt språk.
+  + Information om tjänsterna som du använt hos oss, tidsåtgång, vilka etjänster och funktioner du har använt.
+
+Den rättsliga grunden för behandling av teknisk information enligt ovan är avtal med den registrerade.
+
+
+**Allmänna handlingar och bevarandetid**
+Tänk på att handlingar och uppgifter som du skickar in till Staden via appen blir allmänna handlingar här. Vem som helst kan begära att få ta del av dessa. Vi gör alltid en sekretessprövning enligt offentlighets- och sekretesslagen innan vi lämnar ut handlingar som innehåller personuppgifter. [Läs mer om allmänna handlingar.](https://helsingborg.se/kommun-och-politik/kontakt-och-paverkan/diarium-handlingar-och-protokoll/allmanna-handlingar/)
+Helsingborgs stad följer arkivlagen. Det finns dokumenthanteringsplaner hos nämnderna som anger vad som gäller om gallring. Där framgår om och när handlingar, som kan innehålla personuppgifter, får gallras (slängas).
+
+
+
+**Dina rättigheter**
+Du har rätt att få tillgång till de personuppgifter som rör dig och som finns hos förvaltningarna inom Helsingborgs stad. Den informationen får du i form av ett så kallat registerutdrag som beställs via helsingborg.se.
+Du kan under vissa förutsättningar dessutom ha rätt att
+  + få dina uppgifter raderade
+  + få felaktiga uppgifter rättade
+  + komplettera ofullständiga personuppgifter
+  + begära en begränsning av vår behandling av dina personuppgifter
+  + invända mot vår behandling av dina personuppgifter
+  
+[Läs mer om hur du utövar dina rättigheter och om hur Helsingborgs stad behandlar personuppgifter.](https://helsingborg.se/toppmeny/om-webbplatsen/sa-har-behandlar-vi-dina-personuppgifter/)
+Om du vill komma i kontakt med Stadens dataskyddsombud ring 042-10 50 00 (växel) eller mejla till dataskyddsombud@helsingborg.se 
+
+ 
+
+Om du vill klaga på vår behandling av dina personuppgifter kan du vända dig till Datainspektionen, [http://www.datainspektionen.se/](http://www.datainspektionen.se/)
+`;
+
```

Comment:
Finns det ingen plats där vi har liknande texter samlade? Någon form av content object, ser detta som statiskt innehåll som inte påverkar komponenten mer än att den ska ta emot texten?

Reply:
Yeah, this very text is actually already in a file, no idea why I copied it into the story. Perhaps because I felt it was unclear to use an imported string? But it's polluting the code, so I'll use the imported text instead.